tests/integration/credential/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# This YAML file contains Ansible tasks for managing projects.
+# The tasks include generating a test ID, creating a project, updating the project, and deleting the project.
+
+- name: Generate a random_string for the test
+  set_fact:
+    random_string: "{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+  when: random_string is not defined
+
+- name: Generate a ID for the test
+  set_fact:
+    test_id: "{{ random_string | to_uuid }}"
+  when: test_id is not defined
+
+- name: configure scm_url
+  set_fact:
+    scm_url: "https://github.com/Alex-Izquierdo/eda-sample-project"
```

Comment:
We just migrated this repo to ansible org: https://github.com/ansible/eda-sample-project

I won't block this PR, but please update this in a follow up PR.